qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -251,9 +256,19 @@ def test_create_get_update_delete_w_tenant(self):
         self.assertEqual(data['owner'], 'admin')
         self.assertEqual(data['tenant'], '')
```

Comment:
It would be nice to also add:
```
        self.assertEqual(data['bucket'], 'teuth-test-bucket')
        self.assertEqual(data['bid'], 'teuth-test-bucket')
```

Reply:
Added. Thanks!